### PR TITLE
[93X] Tracker Maps for APV gain payload inspector

### PR DESCRIPTION
**Pull request by mmusich**
Profiting of the technology made available in #19274, we include several types of Tracker Maps of Strip APV gains to be displayed in the Payload Inspector application of the Condition Database Browser @franzoni @arunhep 

attn: @dimattia

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

**Comment by ggovi** [comment]
+1

File: CondCore/SiStripPlugins/interface/SiStripPayloadInspectorHelper.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#ifndef CONDCORE_SISTRIPPLUGINS_SISTRIPPAYLOADINSPECTORHELPER_H
+#define CONDCORE_SISTRIPPLUGINS_SISTRIPPAYLOADINSPECTORHELPER_H
+
+#include <vector>
+#include <numeric>
+
+std::pair<float,float> getTheRange(std::map<uint32_t,float> values){
+  
+  std::vector<float> v;
```
Comment (davidlange6) [inline CondCore/SiStripPlugins/interface/SiStripPayloadInspectorHelper.h]: hi @mmusich - please rearrange this to avoid the vector copy of the map.

Comment (mmusich) [inline CondCore/SiStripPlugins/interface/SiStripPayloadInspectorHelper.h]: hi @davidlange6, copy is avoided. Thanks

**Comment by davidlange6** [comment]
please test

**Comment by mmusich** [comment]
@ggovi can you please sign again? last update doens't change the output that you've been so kind to test beforehand in the [DEMO area of the condDB browser](https://cms-conddb-dev.cern.ch/cmsDbBrowser/payload_inspector/Prod/DEMO/eyJ0YWciOiJTaVN0cmlwQXB2R2Fpbl9Gcm9tUGFydGljbGVzX0dSMTBfdjNfb2ZmbGluZSIsInBsb3QiOiJwbG90X1NpU3RyaXBBcHZHYWluc1JhdGlvTWF4RGV2aWF0aW9uV2l0aFByZXZpb3VzSU9WVHJhY2tlck1hcCIsInBsdWdpbiI6InBsdWdpblNpU3RyaXBBcHZHYWluX1BheWxvYWRJbnNwZWN0b3IiLCJ0eXBlIjoiSW1hZ2UiLCJpb3ZzIjp7InN0YXJ0X2lvdiI6IjEiLCJlbmRfaW92IjoiMTk0MTE1In19)

Posting here a random plot can get be obtained from there:

test

**Comment by ggovi** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**